src/components/widgets/PAutoComplete.js

```diff
@@ -1,26 +1,71 @@
-import { PInput, PListItem, PList } from '../elements'
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { PInput, PListItem, PList, PLoading, PNoData } from '../elements'
+
 export default (props) => {
+  const {
+    value = '',
+    width,
+    size,
+    suggestions = [],
+    loading,
+    onChange = () => null,
+    onSelect = () => null
+  } = props
+
+  const [interalValue, setInternalValue] = useState(value)
```

Comment:
Using internal value lead to losing possibility of changing value programatic from other component. For example, I add some tags for hot keywords. When I click on a tag, the tag will be auto-filled into autocomplete input.